### DISCUSSION TITLE
Unable to open organization owned codespaces

### DISCUSSION THREAD

**Discussion by jtagaca** [community/community#35256] [Codespaces]
Hi I think there is an issue with using the github codespaces in visual studio code in the desktop. It appears that since the codespace is not owned by me and only owned through MLSA organization I am not able to open the codespaces in the visual studio desktop.
 
Can this be looked at? I looked at many sources and it appears as though that is the root cause.
https://drive.google.com/file/d/1Nk8XaBCEmGFu-y5N8W2dQiPZgdAJXCFT/view?usp=sharing
 
Is there anyway a solution can be provided so I can open the codespace in my visual studio code?
Thank you!

**Comment by adnamalin** [comment]
👋  Org ownership should not affect being able to open a codespace in VSCode. Can you confirm you've gone through [these steps to set up the Codespaces VSCode extension](https://docs.github.com/en/codespaces/developing-in-codespaces/using-github-codespaces-in-visual-studio-code#prerequisites).

**Reply by jtagaca** [reply]
That worked thanks so much!!
